physionet-django/console/views.py

```diff
@@ -1590,7 +1590,7 @@ def search_training_applications(request, display_training):
 
 
 @login_required
-@user_passes_test(is_admin, redirect_field_name='project_home')
+@user_passes_test(has_change_credentialapplication_permission, redirect_field_name='project_home')
```

Comment:
Should this be checking something like `user.change_training_permission`?

Reply:
Thank you. resolving Comments in favour of opening another PR for trainings(After discussion with Tom)